src/app/iotHub/models/sharedAccessSignatureAuthorizationRule.ts

```diff
@@ -2,10 +2,11 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License
  **********************************************************/
+import { AccessRights } from './accessRights';
 
 export interface SharedAccessSignatureAuthorizationRule {
-    accessRights: string;
     keyName: string;
-    primaryKey: string;
-    secondaryKey: string;
+    primaryKey?: string;
```

Comment:
can keys be optional?

Reply:
pulled this from hub -- believe they are optional.